tests/functional/tests.py

```diff
@@ -85,21 +86,29 @@ def main():
     newfile_f = 'newfile-f 新'
     newfile_f_custom = 'newfile-f-custom'
 
-
+    # Create a test file
     with open(testfile, 'wb') as file_data:
         file_data.write(fake.text().encode('utf-8'))
     file_data.close()
 
+    # Create a large file
+    with open(largefile, 'wb') as file_data:
+        for i in range(0, 104857):
+            file_data.write(fake.text().encode('utf-8'))
```

Comment:
Just use zero bytes instead of using `fake.text()` would be faster.. and avoids CPU computation.